ckan/plugins/__init__.py

```diff
@@ -1,6 +1,21 @@
 # encoding: utf-8
+import warnings
 
 from ckan.plugins.core import *  # noqa: re-export
 from ckan.plugins.interfaces import *  # noqa: re-export
 
-from ckan.plugins import toolkit  # noqa: re-export
+
+def __getattr__(name):
+    if name == 'toolkit':
+        import ckan.plugins.toolkit as t
+        from ckan.exceptions import CkanDeprecationWarning
+        msg = ("`toolkit` attribute of the `ckan.plugins` module will be"
+               "removed in future CKAN release. Import `ckan.plugins.toolkit`"
+               " directly instead:\n\t"
+               "import ckan.plugins.toolkit as toolkit")
```

Comment:
I don't think this code is hurting anything. do we need to remove it in the future?

Reply:
Nope. I'm just feeling guilty for adding a new hack into core :laughing:  I'll drop this warning